glslc/test/include.py

```diff
@@ -125,12 +132,14 @@ class VerifyCompileIncludeTwoSiblings(expect.ValidObjectFile):
 
     environment = Directory('.', [
         File('b.vert',
-             """#include "a"
+             """#version 140
+             #include "a"
              void bfun(){afun();}
              #include "c"
              """),
         File('a',
-             """void afun(){}
+             """
```

Comment:
Why was this newline necessary?  Is there a another latent bug in the `#include`d content scanner?